### DISCUSSION TITLE
Do you have Discord?

### DISCUSSION THREAD

**Discussion by littlebirdiegr** [activexray/Doplarr#25] [General]
Is there a Discord server for Doplarr?

**Comment by activexray** [comment]
Yes! Here https://discord.gg/YSga2wxr

**Reply by arpitgupta** [reply]
gives me invite invalid. Also the link in readme does not work either.

**Reply by activexray** [reply]
@arpitgupta Good catch! I'm updating to a non-expiring link now.

**Comment by ciordia9** [comment]
Current discord link says I can't join. "Unable to accept invite."
Never been banned anywhere, tried this through web & app.

**Reply by activexray** [reply]
Should be fixed now
